lib/elasticband/filter.rb

```diff
@@ -53,6 +59,12 @@ def parse_includes_filter(includes_options)
         [Filter::Query.new(Elasticband::Query.parse(*includes_options))]
       end
 
+      def parse_range_filter(range_options)
+        return [] if range_options.blank?
+
+        [Filter::Range.new(range_options.keys.first, range_options.values.first)]
```

Comment:
Because there will be only one in the hash syntax, follow the example:

https://www.elastic.co/guide/en/elasticsearch/reference/1.4/query-dsl-range-filter.html